Directory.Build.props

```diff
@@ -23,7 +23,7 @@
 
   <PropertyGroup Condition="$(SIXLABORS_TESTING_PREVIEW) == true">
     <!-- Workaround various issues bound to implicit language features. -->
-    <LangVersion>preview</LangVersion>
+    <LangVersion>12.0</LangVersion>
```

Comment:
In my local setup I had to set this in the csproj to fix the build. I assume this is also somehow broken on main since the ColorMatrix code is untouched in the PR?

Reply:
Yeah, I need to update to 13 there though for .NET 9

Reply:
Do we need .NET 9 for 3.1.x?

Reply:
Thanks for the clarification! I think the solution is to set the version to 12 unconditionally, see #2896.